### PR TITLE
Update the expected plugin file size range

### PR DESCRIPTION
**Pull request by costin-zaharia-sonarsource**

### CODE REVIEW

File: sonar-csharp-plugin/pom.xml [hunk 1]
```diff
@@ -213,8 +213,8 @@
             <configuration>
               <rules>
                 <requireFilesSize>
-                  <maxsize>5400000</maxsize>
-                  <minsize>5200000</minsize>
+                  <maxsize>5410000</maxsize>
+                  <minsize>5300000</minsize>
```
Comment (pavel-mikula-sonarsource) [inline sonar-csharp-plugin/pom.xml:217]: We're bumping the higher digit, see #5447
```suggestion
                <maxsize>5500000</maxsize>
                  <minsize>5300000</minsize>
```

**Review by pavel-mikula-sonarsource — CHANGES_REQUESTED** [review]

File: sonar-vbnet-plugin/pom.xml [hunk 2]
```diff
@@ -215,8 +215,8 @@
             <configuration>
               <rules>
                 <requireFilesSize>
-                  <maxsize>4300000</maxsize>
```
[the remark this answers is not in the document]
Comment (costin-zaharia-sonarsource) [reply sonar-vbnet-plugin/pom.xml:218]: We had some small changes on the VB side and also dependencies updates that will increase the total size of both plugins.

[the remark this answers is not in the document]
Comment (andrei-epure-sonarsource) [reply sonar-vbnet-plugin/pom.xml:218]: hm did we add more dependencies to the analyzers?

[the remark this answers is not in the document]
Comment (pavel-mikula-sonarsource) [reply sonar-vbnet-plugin/pom.xml:218]: There were dependabot updates to existing dependencies (probobuf-java) that might have changed the JAR size

[the remark this answers is not in the document]
Comment (pavel-mikula-sonarsource) [reply sonar-vbnet-plugin/pom.xml:218]: We did not change the list of dependencies, nothing was added

[the remark this answers is not in the document]
Comment (costin-zaharia-sonarsource) [reply sonar-vbnet-plugin/pom.xml:218]: There is one more thing, all the code we add in the common assemblies will go also to the VB and increase the size although it's not always used. Not necessarily a problem, just that we need to be aware of this.

**Pull request merged**